src/components/Input/styles.ts

```diff
@@ -0,0 +1,40 @@
+import {ColorsType} from 'styled-components';
+import styled from 'styled-components/native';
+import Text from '../Text';
+import {BorderProps, InputProps} from './types';
+
+export const Input = {
+  Container: styled.View``,
+  InputInternal: styled.TextInput<{textColor?: ColorsType}>`
+    flex: 1;
+    font-size: ${({theme}) => theme.typography.body3.fontSize}px;
+    font-family: ${({theme}) => theme.typography.body3.fontFamily};
+    color: ${({theme, textColor}) =>
+      theme.colors[textColor] || theme.colors.black};
+    min-height: 50px;
+  `,
+  Border: styled.View<BorderProps>`
+    padding: 0 15px;
+    background-color: ${({color}) => color};
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    border-radius: 10px;
+    border-color: ${({borderColor}) => borderColor};
+    border-width: 1px;
+  `,
+  Error: styled(Text)`
+    margin-top: ${({theme}) => theme.spacing.sm}px;
+  `,
+  Label: styled(Text)`
+    margin-top: ${({theme}) => theme.spacing.sm}px;
+  `,
+  IconContainer: styled.View<Pick<InputProps, 'iconPosition'>>`
+    padding-left: ${({iconPosition}) => {
+      return iconPosition === 'right' ? '10' : '0';
+    }}px;
+    padding-right: ${({iconPosition}) => {
+      return iconPosition === 'left' ? '10' : '0';
+    }}px;
```

Comment:
If you are returning something after the arrow function, you can just remove the 'return' and return the value after the arrow function, without use { } 

```suggestion
    padding-left: ${({iconPosition}) => iconPosition === 'right' ? '10' : '0'}px;
    padding-right: ${({iconPosition}) => iconPosition === 'left' ? '10' : '0'}px;
```

Reply:
solved in c3336cc7837d319e35c2f400b5057b8470ce6e81